### DISCUSSION TITLE
How to update a graph with new documents ?

### DISCUSSION THREAD

**Discussion by tonypius** [microsoft/graphrag#477] [General]
I've created the graph with 100 documents. Now if i need to add say 10 more documents, how can i do it ? Will i need to re-index the entire graph ?

**Comment by SkywalkerDarren** [comment]
Checkout this  discussion --- https://github.com/microsoft/graphrag/discussions/354

**Reply by TheBugKing** [reply]
the new index is created in a new folder with timestamp, when we have to query who would I know which index or artifact to load the data from for response

**Comment by natoverse** [comment]
Tracking incremental ingest with #741
